imap_processing/cdf/config/imap_ultra_global_cdf_attrs.yaml

```
@@ -0,0 +1,151 @@
+Descriptor: &desc
+  Ultra>IMAP Ultra-Energy (IMAP-Ultra) Energetic Neutral Atom Imager
+TEXT: &txt >
+  Ultra captures images of very energetic neutral atoms, particularly hydrogen (H) atoms,
+  produced in the solar system at the heliosheath, the region where the solar wind slows,
+  compresses, and becomes much hotter as it meets the interstellar medium (ISM).
+  The instrument also measures the distribution of solar wind electrons and protons, and
+  the magnetic field. See https://imap.princeton.edu/instruments/ultra for more details.
+
+imap_ultra_l1b_45sensor-de:
+    Descriptor: 45sensor-de
```

I think this is suppose to be this in this line and in others as well. @maxinelasp is that right?
```suggestion
    Descriptor: *desc
```